Sources/Datadog/FeaturesIntegration/CrashReporting/CrashReportingWithRUMIntegration.swift

```diff
@@ -101,6 +101,7 @@ internal struct CrashReportingWithRUMIntegration: CrashReportingIntegration {
         } else {
             // We know it is too late for sending RUM view to previous RUM session as it is now stale on backend.
             // To avoid inconsistency, we only send the RUM error.
+            userLogger.debug("Report the crash as error.")
```

Comment:
How about using present continuous tense? I think this will be more unified with other messages we add here. 
```suggestion
            userLogger.debug("Sending crash as RUM error.")
```